packages/react/src/primitives/shared/utils.ts

```diff
@@ -27,18 +22,14 @@ export const getConsecutiveIntArray = (start: number, end: number) => {
 
 /**
  * Create a uuid to use with amplify fields unless
- * an id is already defined
- * @param id user specified id or undefined
+ * an id is provided
+ * @param id user specified id
  * @returns string
  */
-export const useAmplifyFieldID = (id?: string) => {
-  return React.useMemo(() => {
-    if (id) {
-      return id;
-    }
-    return `amplify-field-${nanoid()}`;
-  }, [id]);
-};
+export const useAmplifyID = (id?: string) => useId(id);
```

Comment:
suggestion: what `Amplify ID` means? would you be open to rename it to `useStableId` or `useProviderId` or something more meaningful?

Reply:
I like `useStableId` a lot, will update.